components/admin/home/create-event-popup.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
+import dayjs from 'dayjs'
+import { ref } from 'vue'
+import { useMutation, useQueryClient } from '@tanstack/vue-query'
+import type { Event } from '~/shared/types'
+
+const props = defineProps<{
+  showPopup: boolean
```

Comment:
Having a component with Popup in the name have a prop called `showPopup` is redundant because popup is in the name. A better naming to consider would be say `visible`